orderer/common/multichannel/registrar_test.go

```diff
@@ -634,8 +640,7 @@ func TestRegistrar_JoinChannel(t *testing.T) {
 		setup(t)
 		defer cleanup()
 
-		ledgerFactory.Close()
-		ledgerFactory, _ = newLedgerAndFactory(tmpdir, "sys-raft-channel", genesisBlockSysRaft)
+		_ = newLedger(ledgerFactory, "sys-raft-channel", genesisBlockSysRaft)
```

Comment:
```suggestion
		newLedger(ledgerFactory, "sys-raft-channel", genesisBlockSysRaft)
```
If we just want to create the ledger and append the block, why the "_ ="?

Reply:
done